src/main/java/edu/kis/powp/jobs2d/features/MouseFeature.java

```diff
@@ -0,0 +1,62 @@
+package edu.kis.powp.jobs2d.features;
+
+import javax.swing.JPanel;
+import javax.swing.event.MouseInputListener;
+
+import edu.kis.powp.appbase.Application;
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.drivers.DriverManager;
+
+public class MouseFeature {
+    private JPanel panel;
+    private Job2dDriver driver;
+
+    private final int LEFT_BUTTON = 1;
+    private final int RIGHT_BUTTON = 3;
+
+    public MouseFeature(DriverManager driverManager, Application application) {
+        this.panel = application.getFreePanel();
+        this.driver = driverManager.getCurrentDriver();
```

Comment:
Isn't there a risk that the current application driver may change? Maybe, it would be better to store driverManager.